lib/bitstyles_phoenix/component/button.ex

```diff
@@ -124,6 +210,132 @@ defmodule BitstylesPhoenix.Component.Button do
       """
   ''')
 
+  story("Small Danger", '''
+      iex> assigns = %{}
+      ...> render ~H"""
+      ...> <.ui_button type="submit" variant="danger" size="small">
+      ...>   Save
+      ...> </.ui_button>
+      ...> """
+      """
+      <button type="submit" class="a-button a-button--danger a-button--small">
+        Save
+      </button>
+      """
+  ''')
+
+  story("X-small Danger", '''
+      iex> assigns = %{}
+      ...> render ~H"""
+      ...> <.ui_button type="submit" variant="danger" size="x-small">
+      ...>   Save
+      ...> </.ui_button>
+      ...> """
+      """
+      <button type="submit" class="a-button a-button--danger a-button--x-small">
+        Save
+      </button>
+      """
+  ''')
+
+  story(
+    "Square Button with Icon",
+    '''
+        iex> assigns = %{}
+        ...> render ~H"""
+        ...> <.ui_icon_button icon="plus" shape={["square", "round"]} label="Add"/>
+        ...> """
+        """
+        <button type="button" class="a-button a-button--square a-button--round" title="Add">
+          <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" class="a-icon a-icon--m" focusable="false" height="20" width="20">
```

Comment:
Should the size 20x20 and the `a-icon--m` really be a default, or is that just by coincidence the case on bistyles itself? cc @planktonic 

I'd personally prefer to keep it empty. Users can still add the `a-icon--m` class via the icon options if they wanted to.

Reply:
Yes I think it’s a good approach to not apply a size unless the user does that explicitly — the default icon size fits the default button size. (I’m working on icons over in bitstyles right now, so I’ll add “remove the size on buttons’ icons” to the list of tasks there 👍 )